config/initializers/0_glitterconfig.rb

```diff
@@ -42,3 +42,5 @@
 elsif Rails.env.production?
 	Glitter::Application.config.repo_dir="ENV['OPENSHIFT_DATA_DIR']/data"
 end
+
+ACCEPTED_FILE_FORMATS = ["image/jpeg","image/png","image/svg+xml","image/gif"]
```

Comment:
Sorry I wasn't completely clear earlier - I meant that the list of  supported formats should be part of the Glitter application config hash - `Glitter::Application.config.accepted_formats = [..]`